content/en/ebpf.md

```diff
@@ -0,0 +1,40 @@
+---
+title: eBPF
+status: Completed
+category: architecture
+---
+
+## What it is
+
+eBPF, or extended Berkeley Packet Filter, is a technology that allows small, sandboxed programs or scripts to run in the kernel space of a Linux system without having to change the kernel's source code or load Linux kernel modules.
+
+A Linux system has two spaces: the kernel and the user space. 
+The kernel represents the operating system's core and is the only part 
+with unlimited access to the hardware. 
+
+Applications reside in the user space, and when they need higher permissions, 
+they send a request to the kernel.
+For applications that require more flexibility, such as direct hardware 
+access, the kernel can be extended via what is known as the "Linux 
+kernel modules" approach. This approach extends the kernel's default functionality
```

Comment:
Question: reading through "the problem it addresses" it seems like eBFP is not a "Linux kernel model", but an alternative...or am I getting this wrong? If so, we could probably say below: "However, this approach also introduces security risks making eBFP an attractive alternative."

Reply:
yes.

Reply:
Applied this as well.